File: ex.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('What is your number? ', (answer) => {
  var num = null
  try {
    var num = parseInt(answer) 
  } catch (error) {
    console.error(error)
    process.exit(0)
  }
  if ((answer % 2 ) === 0) {
      console.log("This is even")
  }else {
    console.log("This is odd")
  }
  console.log(num);
  process.exit(0)
});

File: ex.py
name = raw_input("Give me a number ")
try:
    num = int(name)
except:
    print("You dont know NaN")
    exit()
if (num % 2) == 0:
    print("This is even")
elif (num % 2) == 1:
    print("This is odd")